packages/rocketchat-importer-hipchat-enterprise/server/importer.js

```diff
@@ -282,7 +287,7 @@ Importer.HipChatEnterprise = class ImporterHipChatEnterprise extends Importer.Ba
 			this.collection.update({ _id: this.users._id }, { $set: { 'users': this.users.users }});
 
 			//Import the channels
-			super.updateProgress(Importer.ProgressStep.IMPORTING_CHANNELS);
+			super.updateProgress(ProgressStep.IMPORTING_CHANNELS);
```

Comment:
Curious why super.updateProgress?  Taking a look at the slack one its using just this.updateProgress

Reply:
Great question, although that's been like that since when it was developed...I didn't change it in this pull request.

Reply:
:grin: I mean its valid.  But its inconsistent.  In the cases where the name is the same as the class its extending.. makes complete sense because it can't call this.methodName.